.github/new_pipeline/pull_request_template.md

```diff
@@ -0,0 +1,71 @@
+---
+name: New pipeline
+description: Initiate the peer review process for a new pipeline
+title: "[NEW PIPELINE]: "
+labels: ["pipeline", "user contributed", "peer review needed"]
+---
+
+> [!IMPORTANT]
+> To facilitate the peer review of the new pipeline, do not change the structure of this
+> document. Only the parts in comments should be replaced.
+
+## General information about the pipeline
+
+**Title:** <!-- TITLE GOES HERE -->
+
+| Author | Affiliation | GitHub ID | ORCID | [CRediT Role][role] |
+|-------|-----|----|----|---|
+| | | | | |
+
+[role]: https://credit.niso.org/
+
+> [!INFO]
+> Abstract of the pipeline goes here
+
+## Code information
+
+**GitHub repo:** <!-- REPO URL -->
+
+**LICENSE:** <!-- LICENSE NAME HERE -->
```

Comment:
To remove: This is per script/pipeline, and there could be different licenses in the same PR depending on IP constraints.